unit/dom_test.c

```diff
@@ -93,12 +99,12 @@ int main(int argc, char **argv)
 			free(hints->fabric_attr->prov_name);
 			hints->fabric_attr->prov_name = strdup(optarg);
 			break;
+		case '?':
```

Comment:
I think this was supposed to be `'h'`.  The `'?'` should just be part of the default case.